src/scss/components/optional/_toast.scss

```diff
@@ -12,7 +12,7 @@
 $toast-radius                           : 8px;
 $toast-textColor                        : $white;
 $toast-bgColor--default                 : $grey--darkest;
-$toast-bgColor--alert                   : $red--dark;
+$toast-bgColor--alert                   : $orange--aa;
```

Comment:
Out of interest what does `aa` refer to? Is this the accessibility version?

Reply:
Yep – `aa` is to signify that that is the fully accessible orange. All the new [oranges are here](https://github.com/justeat/fozzie-colour-palette/blob/master/src/scss/index.scss#L11-L24) but essentially we now have:

`$brand--orange` = Completely inaccessible brand orange. Only to be used for highlights (not text or a background for text).
`$orange` = Alternative brand orange which is accessible when used with fonts 18px and higher.
`$orange--aa` = fully accessible murky orange. Can be used if we need an orange colour for text smaller than 18px or as a background colour for smaller text (like here)